README.md

```diff
@@ -1 +1,1146 @@
+
+```   
+   $docker run -d -p 9090:9090 ballerina.guides.io/message_transformation:v1.0
+```
+
+- Here we run the docker image with flag`` -p <host_port>:<container_port>`` so that we use the host port 9090 and the container port 9090. Therefore you can access the service through the host port. 
+
+- Verify docker container is running with the use of `` $ docker ps``. The status of the docker container should be shown as 'Up'. 
+
+- You can access the service using the same curl commands that we've used above. 
+ 
+```
+curl -v http://localhost:9090/contentfilter -d '{"id" : 105, "name" : "ballerinauser", "city" : "Colombo 03", "gender" :    "male"}' -H "Content-Type:application/json" -X POST
+```
+
+### Deploying on Kubernetes
+
+- You can run the service that we developed above, on Kubernetes. The Ballerina language offers native support for running a ballerina program on Kubernetes, with the use of Kubernetes annotations that you can include as part of your service code. Also, it will take care of the creation of the docker images. So you don't need to explicitly create docker images prior to deploying it on Kubernetes. Refer to [Ballerina_Kubernetes_Extension](https://github.com/ballerinax/kubernetes) for more details and samples on Kubernetes deployment with Ballerina. You can also find details on using Minikube to deploy Ballerina programs. 
+
+Since this guide requires MySQL as a prerequisite, you need a couple of more steps to create a MySQL pod and use it with our sample.  
+
+First, let's look at how we can create a MySQL pod in kubernetes. If you are working with minikube, it will be convenient to use the minikube's in-built docker daemon and push the MySQL docker image we are about to build to the minikube's docker registry. This is because during the next steps, in the case of minikube, the docker image we build for employee_database_service will also be pushed to minikube's docker registry. Having both images in the same registry will reduce the configuration steps.
+Run the following command to start using minikube's in-built docker daemon.
+
+```bash
+minikube docker-env
+```
+    
+   * Navigate to the eip-message-transformation/resources directory and run the below command.
+```
+     $docker build -t mysql-ballerina:1.0  .
+```
+
+   *  Then run the following command from the same directory to create the MySQL pod by creating a deployment and service for MySQL. You can find the deployment descriptor and service descriptor in the `./resources/kubernetes` folder.
+```
+      $kubectl create -f ./kubernetes/
+```
+
+Now we need to import `` ballerinax/kubernetes; `` and use `` @kubernetes `` annotations as shown below to enable kubernetes deployment for the service we developed above. 
+
+##### employee_db_service.bal
+
+```ballerina
+import ballerina/http;
+import ballerina/log;
+import ballerina/mime;
+import ballerina/io;
+import ballerina/mysql;
+import ballerinax/kubernetes;
+
+//connect the student details table
```

Comment:
What are we connecting it to? Same applies for the next comment as well.

Reply:
Done